embedding/csharp/Tizen.Flutter.Embedding/FlutterApplication.cs

```diff
@@ -51,6 +51,11 @@ public class FlutterApplication : CoreUIApplication, IPluginRegistry
         /// </summary>
         protected bool IsTopLevel { get; set; } = false;
 
+        /// <summary>
+        /// The renderer type of Flutter engine.
```

Comment:
The description should match with the C++ embedding's.

Reply:
done